hkm/templates/hkm/base.html

```diff
@@ -184,8 +188,9 @@ <h1 class="login-modal__body_title">{% trans 'Sign up' %}</h1>
     <script src="/static/hkm/js/leaflet.js"></script>
     <script src="/static/hkm/js/clipboard.js"></script>
     <script src="/static/hkm/js/hkm-v2.js"></script>
+    <script src="/static/hkm/js/cookie-consent.js"></script>
 
-    {% if ENABLE_ANALYTICS %}
+      {% if ENABLE_ANALYTICS %}
```

Comment:
The `ENABLE_ANALYTICS` variable is never set?

Reply:
Okay, it's an environment variable. But, what I meant with this, is that the tracker was not reacting to cookie consent anyhow, but I guess it's handled some how differently -- but the handler is missing too.

Reply:
its only spacing but fixed